src/tests/MovieDetails.test.js

```diff
@@ -0,0 +1,51 @@
+// Test file for the MovieDetails component
+import React from 'react'
+import { render, waitForElement } from '@testing-library/react'
+import nock from 'nock'
+import MovieDetails from '../components/MovieDetails'
+
+describe('MovieDetails', () => {
+  const match = {
+    params: {
+      id: 12345
+    }
+  }
+
+  beforeEach(() => {
+    nock('https://api.themoviedb.org/3')
+    .get(`/movie/${match.params.id}/videos?language=en-US&page=1&api_key=${process.env.REACT_APP_API_KEY}`)
```

Comment:
Ok for now but-
1. Should use some utility to build query strings. I think there is something built into nock
2. If there are common parameters to all urls, think about using a shared builder
3. Since this test case only calls one html, can consider using more wildcards to match the url. The downside is we would not test what url is actually being called, but that may be okay depending on what we want to test.